sandy/fy.py

```diff
@@ -357,6 +357,115 @@ def apply_qmatrix(self, zam, energy, decay_data):
         new_data.loc[mask, 'FY'] = fy_calc.values
         return self.__class__(new_data)
 
+    def gls_cov_update(self, zam, e, decay_data, Vy=None, kind='cumulative',
+                       threshold=None, index_columns=None):
+        """
+        IFY covariance matrix performed by GLS for a given energy and zam.
```

Comment:
`Update the prior IFY covariance matrix using the GLS technique described in https://doi.org/10.1016/j.anucene.2015.10.027`